static/app/utils/theme.tsx

```diff
@@ -199,67 +246,52 @@ const lightAliases = {
    */
   progressBackground: colors.gray100,
 
-  /**
-   * Background of alerts
-   */
-  alertBackgroundAlpha: 0.3,
-
-  /**
-   * Background of default badge (mainly used in NavTabs)
-   */
-  badgeBackground: colors.gray200,
-
   /**
    * Overlay for partial opacity
    */
-  overlayBackgroundAlpha: 'rgba(255, 255, 255, 0.7)',
+  overlayBackgroundAlpha: color(colors.surface100).alpha(0.7).string(),
 
   /**
    * Tag progress bars
    */
   tagBarHover: colors.purple200,
   tagBar: colors.gray200,
 
-  /**
-   * Color for badge text
-   */
-  badgeText: colors.white,
-
   /**
    * Search filter "token" background
    */
   searchTokenBackground: {
-    valid: '#E8F3FE',
-    validActive: color('#E8F3FE').darken(0.02).string(),
+    valid: colors.blue100,
+    validActive: color(colors.blue100).opaquer(0.2).string(),
     invalid: colors.red100,
-    invalidActive: color(colors.red100).darken(0.02).string(),
+    invalidActive: color(colors.red100).opaquer(0.2).string(),
   },
 
   /**
    * Search filter "token" border
    */
   searchTokenBorder: {
-    valid: '#B5DAFF',
-    validActive: color('#B5DAFF').darken(0.15).string(),
-    invalid: colors.red300,
-    invalidActive: color(colors.red300).darken(0.15).string(),
+    valid: colors.blue200,
+    validActive: color(colors.blue200).opaquer(0.4).string(),
```

Comment:
Since `blue100` is now semi-transparent, we can just increase its opacity for active state. In light mode, search tokens will look darker, and in dark mode, they will look lighter.

Same point for `invalidActive`.